### PR TITLE
Pass comments through pipes instead of as command line arguments

### PR DESCRIPTION
**Pull request by dalen**
This makes the commenting much more reliable when the plan output is hundreds of lines long. 

Without this I would just get
```
/entrypoint.sh: line 99: jq: Argument list too long
```

But with this change it works as it should.

I also made it cut the error output if it is too long. This can happen when it failed to fetch the status of some resource.

### CODE REVIEW

**Comment by sudomateo** [comment]
Thank you for working on this. This is something that I wanted to get done but didn't find the time to do. I'll test this soon.

**Comment by dalen** [comment]
I've updated this to v0.5.3 now as that still had the same issues with handling plans with many resources.

File: src/terraform_plan.sh [hunk 1]
```diff
@@ -29,6 +29,9 @@ function terraformPlan {
         planOutput=$(echo "${planOutput}" | sed -n -r '/-{72}/,/-{72}/{ /-{72}/d; p }')
     fi
     planOutput=$(echo "${planOutput}" | sed -r -e 's/^  \+/\+/g' | sed -r -e 's/^  ~/~/g' | sed -r -e 's/^  -/-/g')
+
+     # If output is longer than max length (65536 characters), keep last part
+    OUTPUT=$(echo "${planOutput}" | tail -c 65000 )
```
Comment (sudomateo) [inline src/terraform_plan.sh]:
```suggestion
    planOutput=$(echo "${planOutput}" | tail -c 65000 )
```

Comment (dalen) [reply src/terraform_plan.sh]: missed that one, fixed.

**Pull request merged**